public/app/features/panel/panel_directive.ts

```diff
@@ -57,7 +57,7 @@ var panelTemplate = `
   </div>
 `;
 
-module.directive('grafanaPanel', function($rootScope) {
+module.directive('grafanaPanel', function($rootScope, $document, $timeout) {
```

Comment:
Yes. I used to delay the refresh call, but then replaced it with debounce and forgot to remove timeout.